python/idsse_common/test/test_aws_utils.py

```diff
@@ -155,8 +159,8 @@ def test_get_issues(aws_utils: AwsUtils, mock_exec_cmd):
     result = aws_utils.get_issues(
         issue_start=EXAMPLE_ISSUE, issue_end=EXAMPLE_VALID, num_issues=2)
     assert len(result) == 2
-    assert result[0] == EXAMPLE_VALID
-    assert result[1] == EXAMPLE_VALID - timedelta(hours=1)
+    assert result[0] == EXAMPLE_VALID - timedelta(hours=1)
```

Comment:
Is this order change random, or is it because of changes to the mock?

Reply:
A small change in the mock.